Packages/vcs/Lib/vtk_ui/text.py

```diff
@@ -49,34 +158,24 @@ def text_actor(string, fgcolor, size, font):
 
     return actor
 
-def text_dimensions(text, text_prop):
-    ren = vtkTextRenderer()
-    bounds = [0,0,0,0]
-    ren.GetBoundingBox(text_prop, text, bounds)
-    return bounds[1] - bounds[0], bounds[3] - bounds[2]
 
-def baseline_offsets(origin, new_string, text_prop):
+def text_dimensions(text, text_prop, at_angle=0):
     ren = vtkTextRenderer()
```

Comment:
Made it so that the function behaves the way I always meant for it to work; it will by default return the dimensions of text that has not been rotated.